charts/airbyte/values.yaml

```diff
@@ -14,9 +14,9 @@ global:
   # Database configuration override
   database:
     # -- Secret name where database credentials are stored
-    secretName: ""
```

Comment:
i think a general good practice is to leave the default values.yaml as it is, and add a different custom-values.yaml one just with the changes (you could add additional fields here). 
The changes on the templates then should be done in such a way that they support both the default values, and the custom ones (so they don't take for granted the new fields that might have been added form the custom-values.yaml)

Reply:
I agree to an extent, but I preferred not to play around too much with the Helm chart and tried using the available configuration options.